output/output.go

```diff
@@ -88,3 +92,39 @@ func ZipAndExportOutput(sourcePth, destinationZipPth, envKey string) error {
 
 	return ExportOutputFile(tmpZipFilePth, destinationZipPth, envKey)
 }
+
+// RunAndExportOutput runs a command.Model and directs its ouput to a log file
+// and keeps only the last N lines of the output.
+func RunAndExportOutput(cmd *command.Model, outputPath, envkey string, outputLastLinesOfCode int) error {
```

Comment:
For me running is an alien in this output package. What do you think about if we would as the following:
- Create a fn here that exports to a file and prints the last N lines.
- Create a fn in the command package where it is possible to set this new output stuff.

I did not looked around of this, I just would like to open this discussion. So, wdyt?

Reply:
To be honest, I had the same feeling, it is sort of a shortcut putting this function in this package. Your proposal is much better at first glance. I will explore that solution by keeping an eye on the points of the above comment.